kDrive/UI/Controller/Files/File List/FileListViewController.swift

```diff
@@ -562,7 +562,12 @@ class FileListViewController: UIViewController, UICollectionViewDataSource, Swip
 
     func showEmptyView(_ isHidden: Bool) {
         guard (collectionView.backgroundView == nil) != isHidden || headerView?.sortView.isHidden == isHidden else { return }
-        let emptyView = EmptyTableView.instantiate(type: viewModel.configuration.emptyViewType, button: false)
+        var type = viewModel.configuration.emptyViewType
+        if type == .emptyFolder || type == .noShared || type == .noSharedWithMe,
```

Comment:
In case of an empty folder in the select view we display emptyFolderWithCreationRights instead of emptyFolder